### DISCUSSION TITLE
Add support for icons with more than one path (ex. Fontawesome Duotone)

### DISCUSSION THREAD

**Discussion by carlos-sarmiento** [home-assistant/frontend#11266] [Other feature requests]
## The request

The new mechanism for adding custom icons to FA is pretty cool, but since it only supports a single path it cannot be used with icons that take multiple paths (like FontAwesome Pro's duotone). It would be great if there was a way to provide multiple paths (with a related css class) so those icons can be incorporated.

## The alternatives

I don't think there are multiple alternatives for this feature requests.

**Comment by bramkragten** [comment]
It should be possible to still use `iron-icon`, the old way as we added a fallback, providing the icon set could be a bit challenging though because you can't have duplicate custom elements that are needed for defining an iron icon set (`iron-iconset-svg`, `iron-meta`)...

**Comment by carlos-sarmiento** [comment]
There is a rather long conversation on the `hass-fontawesome` repo on the challenges of getting it to work. It would be better if HA could provide a non-hackish way to do this :D
